app/(auth)/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -3,6 +3,7 @@ import { SignIn } from '@clerk/nextjs'
 import { type Metadata } from 'next'
 
 export const metadata: Metadata = {
+	metadataBase: new URL('http://localhost:3000'),
```

Comment:
You should change this (all occurrences) to check the environment, and set the URL for production or dev/testing separately.

Reply:
Would something like this be acceptable to you?

```javascript
metadataBase: new URL(process.env.NEXT_PUBLIC_VERCEL_URL ?? 'http://localhost:3000')
```

Reply:
Yep that's perfect

Reply:
@f3rno64 this has been addressed now.